pkgs/development/python-modules/meraki/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3
+, aiohttp
+, requests
+, setuptools
+, typer
```

```suggestion
, aiohttp
, requests
```

One last nit (not all bindings were used anymore) and then this PR looks good to me now!

---

Committed the suggestion then squashed to one commit. Thanks for the review!